src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -30,6 +31,7 @@ def _gen_secret():
 
     @classmethod
     def init(cls):
+        cls.logger = logging.getLogger('jtw')
```

Comment:
This should probaby read:
```suggestion
        cls.logger = logging.getLogger('jwt')
```
I wonder if `jwt-auth` or `auth_jwt` would be more intuitive?